styles.css

```diff
@@ -7,6 +7,11 @@
 body {
   margin: 0;
   padding: 0;
+<<<<<<< Updated upstream
```

Comment:
Since I already merged your font changes in a separate PR, these lines are causing a conflict. Go ahead and delete lines 10-14 here, and GitHub will merge everything appropriately.

Reply:
I'm glad you like the background. I'll fix the conflicts.